### PR TITLE
issue1038 warning when only default emissivities are used MonoLayerAir.

### LINKED ISSUE #1038: warn users when epsLw_a = default in glazing
It's easy to make mistakes when configuring a `glazing` by forgetting to put a non-default epsLw_a. A warning should be added when the default value is used.

### PR DESCRIPTION
**Pull request by Kristoff6**
Should be pulled to issue1038_defaultepsLwGlazingWarning branch, don't seem to be able to create separate branch.
Documentation to be updated?

closes #1038

### CODE REVIEW

**Comment by Mathadon** [comment]
@Kristoff6 looks good on the technical side! I propose the following changes with respect to coding style.
 - [ ] instead of adding `defaultMaterial`, create the package `IDEAS.Buildings.Data.Constants` and add `epsLw_default=0.85`, use this default value in `IDEAS.Buildings.Data.Interfaces.Material` and in your check.
 - [ ] avoid long lines of Modelica code, split them across multiple lines
 - [ ] in variable names, start with a small letter after an underscore, e.g. `m_flow_nominal`, not `m_flow_Nominal`.
 - [ ] throw an error instead of a warning
 - [ ] include an option in the glazing record to disable the error. This may mean that it's more convenient to put the assert in a different model than `MonoLayerAir`.
 - [ ] rather than stating that 'Default emissivities are used', explain why that is a problem. I.e. that the user is probably unintentionally simulating low performance glazing and that the error can be disabled using the parameter. 
 - [ ] Include `"In " + getInstanceName() +": ..."`
 - [ ] Include documentation about the parameter to disable the error in the glazing record.
 - [ ] Update the revision history for changed files/models.
 - [ ] The validation model throws warnings but we cannot automatically check that these warnings are only thrown when required, and that they are not thrown when not required. So the validation model can probably be removed since it does not validate anything (automatically using unit tests).
 - [ ] Each variable and model should have a comment (shortly) explaining the variable/model.

File: IDEAS/RunUnitTests.py [hunk 1]
```diff
@@ -29,6 +29,7 @@
 tester=r.Tester()
 
 """ number of parallel processes started."""
+tester.setSinglePackage("IDEAS.Buildings")
```
Comment (Mathadon) [inline IDEAS/RunUnitTests.py]: please do not commit changes to `RunUnitTests.py`

File: IDEAS/Buildings/Data/Interfaces/Glazing.mo [hunk 2]
```diff
@@ -55,10 +56,18 @@ to reflect the effect of the glazing coating, assuming it exists.
 This makes the difference of a U value of about 3 W/mK or 1 W/mK for double glazing!
 See <a href=\"modelica://IDEAS.Buildings.Data.Glazing.Ins2Ar\">IDEAS.Buildings.Data.Glazing.Ins2Ar</a>
 for an example with double glazing that has 1 glazed sheet with a coating on the inside.
+<p> When no coatings are detected, ie. when the emissivities are default, an error is thrown to inform
```
Comment (Mathadon) [inline IDEAS/Buildings/Data/Interfaces/Glazing.mo]: You add the start of a paragraph using `<p>` but not its end. Also, you inserted a paragraph within another paragraph.

File: IDEAS/Buildings/Data/Constants.mo [hunk 3]
```diff
@@ -0,0 +1,7 @@
+within IDEAS.Buildings.Data;
+package Constants
```
Comment (Mathadon) [inline IDEAS/Buildings/Data/Constants.mo]: please add a comment for this package

File: IDEAS/Buildings/Components/Window.mo [hunk 4]
```diff
@@ -22,7 +22,8 @@ model Window "Multipane window"
       mats=glazing.mats,
       energyDynamics=if windowDynamicsType == IDEAS.Buildings.Components.Interfaces.WindowDynamicsType.Normal then energyDynamics else Modelica.Fluid.Types.Dynamics.SteadyState,
       dT_nom_air=5,
-      linIntCon=true));
+      linIntCon=true),
+      errorLowPerformance = errorLowPerformance);
```
Comment (Mathadon) [inline IDEAS/Buildings/Components/Window.mo]: I don't think that we need this parameter here? You can directly set `multiLayer.errorLowPerformance = glazing.errorLowPerformance`

File: IDEAS/Buildings/Components/Interfaces/PartialSurface.mo [hunk 5]
```diff
@@ -56,7 +59,8 @@ partial model PartialSurface "Partial model for building envelope component"
   IDEAS.Buildings.Components.BaseClasses.ConductiveHeatTransfer.MultiLayer
     layMul(final inc=inc, energyDynamics=energyDynamics,
     linIntCon=linIntCon_a or sim.linearise,
-    A=A)
+    A=A,
+    errorLowPerformance = errorLowPerformance)
```
Comment (Mathadon) [inline IDEAS/Buildings/Components/Interfaces/PartialSurface.mo]: I don't think that we need this parameter here? You can directly set multiLayer.errorLowPerformance = glazing.errorLowPerformance

File: IDEAS/Buildings/Components/Interfaces/PartialSurface.mo [hunk 6]
```diff
@@ -32,6 +32,9 @@ partial model PartialSurface "Partial model for building envelope component"
     "Static (steady state) or transient (dynamic) thermal conduction model"
     annotation(Evaluate=true, Dialog(tab = "Dynamics", group="Equations"));
 
+  parameter Boolean errorLowPerformance = true
```
Comment (Mathadon) [inline IDEAS/Buildings/Components/Interfaces/PartialSurface.mo]: I don't think that we need this parameter here? You can directly set multiLayer.errorLowPerformance = glazing.errorLowPerformance without needing this parameter, which the user would then have to worry about.

File: IDEAS/Buildings/Components/BaseClasses/ConvectiveHeatTransfer/package.order [hunk 7]
```diff
@@ -3,4 +3,3 @@ ExteriorConvection
 InteriorConvection
 MonoLayerAir
 Examples
-Validation
```
Comment (Mathadon) [inline IDEAS/Buildings/Components/BaseClasses/ConvectiveHeatTransfer/package.order]: It looks like you removed a package? Watch out when committing 'all' files. It's good practice to manually double-check that you only commit files with changes that are required for the PR.

Comment (Kristoff6) [reply IDEAS/Buildings/Components/BaseClasses/ConvectiveHeatTransfer/package.order]: Yes, I removed the previous stated unnessecary validation package.

File: IDEAS/Buildings/Components/Interfaces/PartialSurface.mo (hunk 6 above)
Comment (Kristoff6) [reply IDEAS/Buildings/Components/Interfaces/PartialSurface.mo]: Ok, great, changed!

**Comment by Mathadon** [comment]
closing this such that I can open a PR myself

**Pull request closed without merging**